test/python/transpiler/test_preset_passmanagers.py

```diff
@@ -820,21 +826,22 @@ def test_layout_tokyo_2845(self, level):
             3: Qubit(QuantumRegister(15, "ancilla"), 3),
             4: Qubit(QuantumRegister(15, "ancilla"), 4),
             5: Qubit(QuantumRegister(15, "ancilla"), 5),
-            6: Qubit(QuantumRegister(3, "qr1"), 1),
-            7: Qubit(QuantumRegister(15, "ancilla"), 6),
-            8: Qubit(QuantumRegister(15, "ancilla"), 7),
+            6: Qubit(QuantumRegister(15, "ancilla"), 6),
+            7: Qubit(QuantumRegister(15, "ancilla"), 7),
+            8: Qubit(QuantumRegister(3, "qr1"), 1),
             9: Qubit(QuantumRegister(15, "ancilla"), 8),
-            10: Qubit(QuantumRegister(3, "qr1"), 0),
-            11: Qubit(QuantumRegister(3, "qr1"), 2),
-            12: Qubit(QuantumRegister(15, "ancilla"), 9),
-            13: Qubit(QuantumRegister(15, "ancilla"), 10),
-            14: Qubit(QuantumRegister(15, "ancilla"), 11),
-            15: Qubit(QuantumRegister(15, "ancilla"), 12),
-            16: Qubit(QuantumRegister(2, "qr2"), 0),
-            17: Qubit(QuantumRegister(2, "qr2"), 1),
-            18: Qubit(QuantumRegister(15, "ancilla"), 13),
-            19: Qubit(QuantumRegister(15, "ancilla"), 14),
+            10: Qubit(QuantumRegister(15, "ancilla"), 9),
+            11: Qubit(QuantumRegister(15, "ancilla"), 10),
+            12: Qubit(QuantumRegister(3, "qr1"), 0),
+            13: Qubit(QuantumRegister(3, "qr1"), 2),
+            14: Qubit(QuantumRegister(2, "qr2"), 1),
+            15: Qubit(QuantumRegister(15, "ancilla"), 11),
+            16: Qubit(QuantumRegister(15, "ancilla"), 12),
+            17: Qubit(QuantumRegister(15, "ancilla"), 13),
+            18: Qubit(QuantumRegister(15, "ancilla"), 14),
+            19: Qubit(QuantumRegister(2, "qr2"), 0),
```

Comment:
What exactly makes the output different here? Is it related to how the seed is being used in the new backend?

Reply:
The `vf2_layout` pass tries to solve a subgraph isomorphism problem. From all the possible answers to this problem, it picks the one that minimizes the gate error, which depends on the backend properties. `FakeTokyo` and `Fake20QV1` share the coupling map (because I inject it manually) but don't have the same backend properties, and this is why the optimal solution is a bit different.